MM2SpaceCenter.roboFontExt/lib/MM2SpaceCenter.py

```diff
@@ -15,139 +15,147 @@
 from defconAppKit.windows.baseWindow import BaseWindowController
 from mojo.events import addObserver, removeObserver, postEvent
 from vanilla import *
+from lib.UI.integerEditText import NumberEditText
 from mojo.extensions import *
 import codecs
 
 class MM2SpaceCenter:
     
     def activateModule(self):
         addObserver(self, "MMPairChangedObserver", "MetricsMachine.currentPairChanged")
+        
         print ('MM2SpaceCenter activated')
 
+
     def deactivateModule(self, sender):
         removeObserver(self, "MetricsMachine.currentPairChanged")
+        removeObserver(self, "FutureProof.currentPairChanged")
+
         print ('MM2SpaceCenter deactivated')
 
-    def __init__(self, ):
-        self.font = metricsMachine.CurrentFont()
-        self.pair = metricsMachine.GetCurrentPair()
-        #self.wordlistPath = wordlistPath
-        
-        
-        
-        
-        leftMargin = 10
-        topMargin = 5
-        yPos = 0 
-        lineHeight = 20
-        
-        yPos += topMargin
-        
-        self.messageText = 'MM2SpaceCenter activated 😎'
 
+    def __init__(self, ):
+        #self.font = metricsMachine.CurrentFont()
+        #self.pair = metricsMachine.GetCurrentPair()
 
+        self.font = CurrentFont()
+        try:
+            self.pair = metricsMachine.GetCurrentPair() 
+        except:
+             self.pair = ('A', 'V')
+        
+        leftMargin = 11
+        topMargin = 0
+        lineHeight = 18
+        columnWidth = 90  
+        second_col_x = columnWidth + leftMargin + 10
 
+        w_w = 200
         
+        self.messageText = '😎 Activated'
+
         self.wordCount = 20
         self.minLength = 3
         self.maxLength = 15
         
         self.activateModule()
-        self.w = Window((250, 155), "MM2SpaceCenter")
+        self.w = Window((w_w, 0), "MM2SpaceCenter")
+        self.w.getNSWindow().setTitlebarHeight_(26)
+        self.w.getNSWindow().setTitlebarAppearsTransparent_(True)
         
-        self.w.myTextBox = TextBox((leftMargin, yPos, -10, 17), self.messageText, sizeStyle="regular") 
-
+        yPos = topMargin
 
+        self.w.statusBar = Box((3, 0, -3, 30))
```

Comment:
Status bar feels like a status hierarchically. Helps to delineate areas in this tight space.